ShareX/ShareX.csproj

```diff
@@ -1037,16 +1036,8 @@
   </PropertyGroup>
   <PropertyGroup>
     <PreBuildEvent>cd $(ProjectDir)
-
-if $(ConfigurationName) == Release (
-    for %25%25x in (git.exe) do if not [%25%25~$PATH:x]==[] (
-        git rev-parse HEAD &gt; GitHash.txt
-    )
-)
-
-if not exist GitHash.txt (
-    type nul &gt; GitHash.txt
-)</PreBuildEvent>
+git &amp;&amp; git rev-parse HEAD &gt; GitHash.txt || echo= &gt;&gt; GitHash.txt
```

Comment:
This command is checking if git exists, if so, run the git command, else append a new line to the GitHash.txt file